### PR TITLE
Added clear and reload

### PR DESCRIPTION
**Pull request by Raining-Cloud**
basically what the title says, i added a reload command which automatically loads the last loaded file again. And clear just clears the console

### CODE REVIEW

File: src/cli/main.rs [hunk 1]
```diff
@@ -35,6 +35,7 @@ fn _main() -> Result<(), i32> {
 
     let io = emulator::IoRegisters::new();
     let mut computer = Computer::new(&io);
+    let mut last_file = String::from("none");
```
Comment (MalteT) [inline src/cli/main.rs:38]: Type `Option<String>` would be more idiomatic

File: src/cli/ui.rs [hunk 2]
```diff
@@ -111,17 +111,23 @@ pub fn display_program(program: &Program) {
     println!();
 }
 
+pub fn clear_console() {
+    print!("\x1B[2J\x1B[1;1H");
```
Comment (MalteT) [inline src/cli/ui.rs:115]: This works on Linux, but does this also work on Windows/Mac?

File: Cargo.lock [hunk 3]
```diff
@@ -1,295 +1,294 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```
Comment (MalteT) [inline Cargo.lock:3]: I guess an update to the lockfile is due, but it should probably not be in the same PR

**Review by MalteT — COMMENTED** [review]
Just my opinion, changes seem to work well on my machine though!

File: src/cli/ui.rs (hunk 2 above)
Comment (Raining-Cloud) [reply src/cli/ui.rs:115]: It works for Windows and Linux can't test for Mac

File: Cargo.lock (hunk 3 above)
Comment (Raining-Cloud) [reply Cargo.lock:3]: No idea why this updated the version tbh

File: src/cli/main.rs (hunk 1 above)
Comment (Raining-Cloud) [reply src/cli/main.rs:38]: I am really not into rust at all, needa check how `Option<String>`  works

File: Cargo.lock (hunk 3 above)
Comment (klemens) [reply Cargo.lock:3]: This is done automatically when you run any cargo command with a new enough rust version. As suggested by Malte, please move this into a separate commit, but you can keep it in the PR.

File: Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "emulator-2i"
-version = "2.0.2"
+version = "2.0.3"
```
Comment (klemens) [inline Cargo.toml:3]: Please undo this, I will create a separate commit bumping the version (see the history).

File: src/cli/main.rs [hunk 5]
```diff
@@ -73,10 +74,11 @@ fn _main() -> Result<(), i32> {
                 println!("Fehler: Kein Mikroprogramm geladen! (Laden per \"load prog.2i\")");
             }
         } else if line.starts_with("load ") {
-            let path = cmdline_parser::parse_single(&line[5..].trim());
+            let main_path = cmdline_parser::parse_single(&line[5..].trim());
```
Comment (klemens) [inline src/cli/main.rs:77]: Why did you rename this variable?

File: src/cli/ui.rs (hunk 2 above)
Comment (klemens) [reply src/cli/ui.rs:115]: I am pretty sure this also works on mac os. Windows only supports it in "recent" versions, but I guess that is good enough. :sweat_smile:

File: src/cli/main.rs [hunk 6]
```diff
@@ -93,8 +95,20 @@ fn _main() -> Result<(), i32> {
...
+                println!("Es wurde noch keine Datei geladen");
+            } else if let Ok(prog) = load_programm(Path::new(&last_file))  {    
+                program = Some(prog);
+                // Reset computer (only keep io registers)
+                computer = Computer::new(&io);
+                ui::status(&mut computer, &io, &program, None);
+            }
         } else if line == "help" {
             ui::display_help();
+        }else if line == "clear" {
```
Comment (klemens) [inline src/cli/main.rs:109]:
```suggestion
        } else if line == "clear" {
```

File: src/cli/main.rs (hunk 5 above)
Comment (Raining-Cloud) [reply src/cli/main.rs:77]: > Why did you rename this variable?

Ah i added some more stuff previously but forgot to change it back

**Comment by Raining-Cloud** [comment]
FIrst pull request, i learned a lot thank you guys! 👍
